MemMrg/memmrg.h

```diff
@@ -0,0 +1,42 @@
+#ifndef MEMMRG_H
+#define MEMMRG_H
+
+
+typedef char Data;
+
+
+class MemMrg
+{
+    struct MrgItem
+    {
+        MrgItem();
+
+        MrgItem *next_;
+        char item_val_[28];
+
```

Comment:
No one garanties that pointer is 4 bytes.
No one garanties that this structure size will be 32 bytes - alligment should be taken into account